PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalClient.kt

```diff
@@ -212,13 +212,6 @@ class PayPalClient internal constructor(
         val switchInitiatedTime = Uri.parse(approvalUrl).getQueryParameter("switch_initiated_time")
         val isAppSwitchFlow = !switchInitiatedTime.isNullOrEmpty()
 
-        if (isAppSwitchFlow) {
```

Comment:
not sure why `HANDLE_RETURN_STARTED` was tied to app switch flow, but I'm copying that logic into `PayPalLauncher`.